### PR TITLE
Updates to validator details for Helvetia Staking

### PR DESCRIPTION
**Pull request by joeleol**
Updated: fee, payoutschedule and contact fields.

### CODE REVIEW

File: public/validators/main-albatross/NQ37 6EL5 BP9K XL1A 3ED0 L3EC NPR5 C9D3 BRKG.json [hunk 1]
```diff
@@ -3,8 +3,14 @@
   "description": "Secure.Reliable.Swiss.",
   "website": "https://helvetia-staking.ch",
   "address": "NQ37 6EL5 BP9K XL1A 3ED0 L3EC NPR5 C9D3 BRKG",
-  "fee": 0.08,
+  "fee": 0.069,
```
Comment (onmax) [inline public/validators/main-albatross/NQ37 6EL5 BP9K XL1A 3ED0 L3EC NPR5 C9D3 BRKG.json:6]: nice

File: public/validators/main-albatross/NQ37 6EL5 BP9K XL1A 3ED0 L3EC NPR5 C9D3 BRKG.json [hunk 2]
```diff
@@ -3,8 +3,14 @@
...
+  "contact": {
+    "telegram": "@joel_eol",
+    "twitter": "@helvetiastaking",
+    "discordInvitationUrl": "https://discord.com/channels/359979114062675969/1286574729989324810",
```
Comment (onmax) [inline public/validators/main-albatross/NQ37 6EL5 BP9K XL1A 3ED0 L3EC NPR5 C9D3 BRKG.json]: I tried to join to test it but I think is invalid.

Can you create the link as you are inviting someone to join? Thank you

you should see something like this:

image

Comment (onmax) [reply public/validators/main-albatross/NQ37 6EL5 BP9K XL1A 3ED0 L3EC NPR5 C9D3 BRKG.json]: Make sure to set the expiration date to never!

File: public/validators/main-albatross/NQ37 6EL5 BP9K XL1A 3ED0 L3EC NPR5 C9D3 BRKG.json [hunk 3]
```diff
@@ -3,8 +3,14 @@
   "description": "Secure.Reliable.Swiss.",
   "website": "https://helvetia-staking.ch",
   "address": "NQ37 6EL5 BP9K XL1A 3ED0 L3EC NPR5 C9D3 BRKG",
-  "fee": 0.08,
+  "fee": 0.069,
+  "payoutSchedule": "Every 360 batches (~6hrs)"
```
Comment (onmax) [inline public/validators/main-albatross/NQ37 6EL5 BP9K XL1A 3ED0 L3EC NPR5 C9D3 BRKG.json]: This text ideally will be shown to many type of users including beginner users.

Would you mind to rename it just to "Every 6 hours" to keep consistency with the rest of the validators?

thank you!

**Review by onmax — CHANGES_REQUESTED** [review]
Thank you!

File: public/validators/main-albatross/NQ37 6EL5 BP9K XL1A 3ED0 L3EC NPR5 C9D3 BRKG.json (hunk 2 above)
Comment (Albermonte) [reply public/validators/main-albatross/NQ37 6EL5 BP9K XL1A 3ED0 L3EC NPR5 C9D3 BRKG.json]: I think that's a link to the channel inside the Nimiq discord server, so the invitation link would be https://discord.gg/nimiq
There's no way to redirect to the channel if the user is not in the discord  server

**Comment by joeleol** [comment]
Done, hopefully ok to merge now. Would be great to get this in pre-migration!

**Comment by onmax** [comment]
Thank you!

**Pull request merged**